vizro-core/src/vizro/__init__.py

```diff
@@ -14,7 +14,7 @@
 
 __all__ = ["Vizro"]
 
-__version__ = "0.1.26.dev0"
+__version__ = "0.1.26.dev1"
```

Comment:
I don't get why this change is made. Just to test the CI or?

Reply:
It was because I wanted the user to pip install this branch to test, and it's easy to get that command wrong so that it doesn't install what you think. This gives a way of checking that the correct version had been installed.